tests/src/common/WskCli.java

```diff
@@ -418,13 +417,13 @@ public String bindPackage(int expectedCode, String packageName, String name, Map
         RunResult result = cli(expectedCode, cmd);
         String stdout = result.stdout;
         if (expectedCode == SUCCESS_EXIT) {
-            assertTrue(stdout, stdout.contains("ok: created binding " + name));
+            assertTrue(stdout, stdout.contains("created binding " + name));
```

Comment:
does the python CLI still return ok?     If so, please make this work even if there's a global switch between python and go.